src/main/java/uk/gov/hmcts/darts/event/config/DarNotifyEventConfiguration.java

```diff
@@ -25,6 +25,7 @@ public HttpComponentsMessageSender httpComponentsMessageSender() {
     public Jaxb2Marshaller marshaller() {
         var marshaller = new Jaxb2Marshaller();
         marshaller.setPackagesToScan("com.viqsoultions");
+        marshaller.setValidationEventHandler(new  jakarta.xml.bind.helpers.DefaultValidationEventHandler());
```

Comment:
without this, any unmarshalling error won't be raised, and unmarshalling silently fails and unmarshalls to `null` field values